model/schema/ontologyTerm.yaml

```diff
@@ -286,9 +285,51 @@ classes:
 
   XPOTerm:
     is_a: PhenotypeTerm
+    description: >-
+      A phenotype ontology term from the Xenopus Phenotype Ontology (XPO).
+      PURL: http://purl.obolibrary.org/obo/xpo.owl
 
   MPTerm:
     is_a: PhenotypeTerm
+    description: >-
+      A phenotype ontology term from the Mammalian Phenotype (MP) Ontology.
+      PURL: http://purl.obolibrary.org/obo/mp.owl
+
+  HPTerm:
+    is_a: PhenotypeTerm
+    description: >-
+      A phenotype ontology term from the Human Phenotype (MP) Ontology.
+      PURL: http://purl.obolibrary.org/obo/hp.owl
+
+  WBPhenotypeTerm:
+    is_a: PhenotypeTerm
+    description: >-
+      A phenotype ontology term from the (WormBase) C. elegans Phenotype Ontology (WBPhenotype).
+      PURL: http://purl.obolibrary.org/obo/wbphenotype.owl
+
+  APOTerm:
+    is_a: PhenotypeTerm
+    description: >-
+      A phenotype ontology term from the Ascomycete Phenotype Ontology (APO).
+      PURL: http://purl.obolibrary.org/obo/APO.owl
+
+  DPOTerm:
+    is_a: PhenotypeTerm
+    description: >-
+      A phenotype ontology term from the Drosophila Phenotype Ontology (DPO).
+      PURL: http://purl.obolibrary.org/obo/dpo.owl
+
+  PATOTerm:
+    is_a: OntologyTerm
```

Comment:
Should this be PhenotypeTerm?  Maybe traits aren't phenotypes but it feels like we probably want to use this ontology in fields that are restricted to phenotype terms.

Reply:
@markquintontulloch  I am not aware of any curators using PATO terms directly for phenotypes. These terms tend to be used to help generate logical definitions for phenotype terms, but are not used directly as phenotype terms

Reply:
...but I can double check with curators

Reply:
cool, we can always change later